[2022-04-07 15:08:30,297] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: s3_uni_flores.upload_to_s3 scheduled__2022-04-06T00:00:00+00:00 [queued]>
[2022-04-07 15:08:30,357] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: s3_uni_flores.upload_to_s3 scheduled__2022-04-06T00:00:00+00:00 [queued]>
[2022-04-07 15:08:30,359] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 15:08:30,359] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-04-07 15:08:30,359] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 15:08:30,727] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): upload_to_s3> on 2022-04-06 00:00:00+00:00
[2022-04-07 15:08:30,731] {standard_task_runner.py:52} INFO - Started process 178 to run task
[2022-04-07 15:08:30,737] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 's3_uni_flores', 'upload_to_s3', 'scheduled__2022-04-06T00:00:00+00:00', '--job-id', '375', '--raw', '--subdir', 'DAGS_FOLDER/dag_s3_universidad_de_las_flores.py', '--cfg-path', '/tmp/tmp_3crc2ln', '--error-file', '/tmp/tmp4nnqe4h_']
[2022-04-07 15:08:30,738] {standard_task_runner.py:77} INFO - Job 375: Subtask upload_to_s3
[2022-04-07 15:08:30,926] {logging_mixin.py:109} INFO - Running <TaskInstance: s3_uni_flores.upload_to_s3 scheduled__2022-04-06T00:00:00+00:00 [running]> on host LAPTOP-5HJG55H7.localdomain
[2022-04-07 15:08:31,498] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=s3_uni_flores
AIRFLOW_CTX_TASK_ID=upload_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2022-04-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-06T00:00:00+00:00
[2022-04-07 15:08:31,508] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-04-07 15:08:38,127] {dag_s3_universidad_de_las_flores.py:27} INFO - filename: /home/lowenhard/airflow/files/ET_Univ_DE_Flores.txt, s3 key: ET_Univ_DE_Flores.txt
[2022-04-07 15:08:39,339] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-04-07 15:08:39,363] {dag_s3_universidad_de_las_flores.py:28} INFO - Created Connection: Session(region_name=None)
[2022-04-07 15:08:39,364] {dag_s3_universidad_de_las_flores.py:29} INFO - File exists True
[2022-04-07 15:08:39,364] {dag_s3_universidad_de_las_flores.py:30} INFO - File exists: True
[2022-04-07 15:08:39,367] {python.py:175} INFO - Done. Returned value was: None
[2022-04-07 15:08:39,378] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=s3_uni_flores, task_id=upload_to_s3, execution_date=20220406T000000, start_date=20220407T200830, end_date=20220407T200839
[2022-04-07 15:08:40,091] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 15:08:40,113] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
